hazelcast/src/main/java/com/hazelcast/nio/tcp/SocketWriter.java

```diff
@@ -18,14 +18,71 @@
 
 import com.hazelcast.nio.SocketWritable;
 
-import java.nio.ByteBuffer;
-
 /**
- * Writes a {@link SocketWriter} to a {@link ByteBuffer}.
+ * Each {@link TcpIpConnection} has a {@link SocketWriter} and it writes {@link SocketWriter} instances to the socket. Copying
+ * the SocketWritable instances to the byte-buffer is done using the {@link WriteHandler}.
```

Comment:
The second link should be `and it writed {@link SocketWritable}` not `{@link SocketWriter}`